perun-auditlogger/src/main/resources/META-INF/spring.handlers

```diff
@@ -0,0 +1,6 @@
+http\://www.springframework.org/schema/tx=org.springframework.transaction.config.TxNamespaceHandler
```

Comment:
Why do we need this file? There is no such file for perun-ldapc. It should be create automatically by maven-shade-plugin.

Reply:
Removed from git.